packages/apps/spaces/ui/form/Select/FormSelect.tsx

```diff
@@ -20,7 +22,16 @@ export const FormSelect = forwardRef<SelectInstance, FormSelectProps>(
 
     return (
       <div className='w-full'>
-        <label {...labelProps}>{label}</label>
+        <label
+          className={cn({
+            absolute: !isLabelVisible,
+            'top-[-999999px]': !isLabelVisible,
```

Comment:
why is this needed? If you really need this somehow you can pass it as `className` through `labelProps` prop.

Reply:
it's needed in address form, adding single prop is cleaner than repeating the whole cn logic. It is used to improve accessability for inputs that have visually hidden labels

Reply:
but setting top: -999999px, is this ok? We did not have this before in the chakra version like this.